### PR TITLE
new icon: sqlite (original, original-wordmark, plain)

### PR DESCRIPTION
**Pull request by Snailedlt**
This is my first venture into the world of SVGs so probably a lot of things that need fixing or improving here. Anyways, leave some feedback, and let me know what I can do better! :)

fixing #437

# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

https://www.sqlite.org/index.html

### CODE REVIEW

File: icons/sqlite/sqlite-plain.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+</g>
+</svg>
```
Comment (Panquesito7) [inline icons/sqlite/sqlite-plain.svg:6]: Missing an endline.
```suggestion
</svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work! 🚀
Could you please create a  `plain-wordmark` version? Should be the same as `original-wordmark` but black. Thanks. 🙂

**Comment by Snailedlt** [comment]
> Great work! 🚀 Could you please create a `plain-wordmark` version? Should be the same as `original-wordmark` but black. Thanks. 🙂

I'll add this. But shouldn't that ideally just be generated automatically once the PR is merged? I mean if the only difference is the color :)

**Comment by Snailedlt** [comment]
@Panquesito7 
Can you take a look again now?
See two latest commits for the updates I did.

File: icons/sqlite/sqlite-plain.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="180"
```
Comment (Panquesito7) [inline icons/sqlite/sqlite-plain.svg:5]: I think the width, height, and viewbox should be `128`? 🤔
If you're having trouble with resizing, try to use [**this tool**](https://www.iloveimg.com/resize-image/resize-svg).

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by Snailedlt** [comment]
> LGTM 🚀 Thank you for your contribution! 😄👍

Thanks for helping! 🙇‍♂️

File: devicon.json [hunk 3]
```diff
@@ -3991,6 +3991,23 @@
...
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#0f80cc"
```
Comment (Panquesito7) [inline devicon.json:4009]:
```suggestion
        "color": "#0f80cc",
        "aliases": []
```

File: devicon.json [hunk 4]
```diff
@@ -3991,6 +3991,24 @@
...
+            "database",
+            "db"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ]
```
Comment (Panquesito7) [inline devicon.json:4007]:
```suggestion
            ],
          "font": [
              "plain",
              "plain-wordmark"
          ]
```

**Comment by Panquesito7** [comment]
Any reasons why it's failing, @Thomas-Boi? 👀

**Comment by Thomas-Boi** [comment]
Due to the bugs related to `peek-bot` and `upload-artifact`, this PR will need longer than usual to be merged. Hopefully, it'll be merged within the next 2 weeks. Until then, thank you for your patience and understanding.

**Review by Thomas-Boi — APPROVED** [review]
The icon looks wonderful! Congratulations on getting your first icon into the repo. Also thank you very much for your patience while we fix the bug.

**Comment by Snailedlt** [comment]
> The icon looks wonderful! Congratulations on getting your first icon into the repo. Also thank you very much for your patience while we fix the bug.

Thanks, happy to help! 💯

**Review by Panquesito7 — COMMENTED** [review]
Awesome! 🚀

**Pull request merged**